templates/miq-template.yaml

```diff
@@ -446,6 +524,8 @@ objects:
             value: "${POSTGRESQL_MAX_CONNECTIONS}"
           - name: POSTGRESQL_SHARED_BUFFERS
             value: "${POSTGRESQL_SHARED_BUFFERS}"
+          - name: POSTGRESQL_CONFIG_DIR
```

Comment:
@carbonin I could see some cases where you could want to supply your own location for your stacked configs (perhaps during troubleshoot/debug), also allow us to dynamically move to whatever directory SCL eventually decide for drop in config-customs. I don't think it hurts. My 2 cents.